anyway/parsers/news_flash_db_adapter.py

```diff
@@ -69,6 +70,8 @@ def insert_new_newsflash(self, newsflash: NewsFlash) -> None:
                      .format(newsflash.accident, newsflash.date))
         self.db.session.add(newsflash)
         self.db.session.commit()
+        infographics_data_cache_updater.add_news_flash_to_cache(newsflash)
+        logging.debug(f"{newsflash.get_id()} added to cache")
```

Comment:
perhaps this debug logging should be info instead?

Reply:
Also, perhaps we shouldn't print it if news flash wasn't added to cache (meaning use return value from add_news_flash_to_cache, and return True when it was added and false otherwise.

Another option which is probably better, is adding this logging of success or failure or no insertion since it's not  eligible inside the add_news_flash_to_cache function

Reply:
OK